tools/qc/fastq_utils/.shed.yml

```diff
@@ -0,0 +1,21 @@
+name: fastq_utils
+owner: ebi-gxa
+description: "scanpy-scripts, command-line wrapper scripts around Scanpy."
```

Comment:
Please change to a suitable description and long description of the whole package (fastq_utils).